lib/schedule/health/checkers/routes_checker.ex

```diff
@@ -11,6 +12,13 @@ defmodule Schedule.Health.Checkers.RoutesChecker do
   def healthy?(%{min_length: min_length}) do
     routes_fn = Application.get_env(:skate_web, :routes_fn, &Schedule.all_routes/0)
 
-    routes_fn.() |> length() >= min_length
+    length = routes_fn.() |> length()
+    passfail = length >= min_length
+
+    if(!passfail) do
+      Logger.warning("Routes Checker failed. min_legth=#{min_length} length=#{length}")
```

Comment:
Optional suggestion: rather than saying "Routes Checker failed", you can use the special `__MODULE__` variable, as in `"#{__MODULE__} checker failed" which will automatically include the full name of the module to make it easy to find the source when someone sees this message.